### DISCUSSION TITLE
[Resolved] UI/UX problem on Vercel.com/.../domains -- can't access or see anything

### DISCUSSION THREAD

**Discussion by zadamg** [vercel/community#2280] [Bug Report]
[placed under bug report because it seems like a UI bug]

I am using vercel's DNS nameservers (domain is on namecheap).  Site is working, but now I can't see the Domains tab for the domain in question. Instead, I get a CSS gradient overlay that covers nearly the entire screen.  No message or anything to indicate why I can't see or access any of my domain settings.  Ad blockers and similar extensions are off.

I have been on Vercel's DNS nameservers for a few days so this isn't some waiting period or anything... I have two questions:

1. is it too much to ask for css-blocking elements in a UI to contain some sort of notice or indication why blocked?  
2. what do I need to do to access my domains panel so I can set-up my email?

Thanks!    Here's what I see...
![image](https://user-images.githubusercontent.com/49826721/235951525-650e2fa7-2540-466e-8d78-60d5b9871185.png)

**Comment by amyegan** [comment]
Thanks for reporting this, @zadamg. I shared it with the team, and we'll update you here when more info is available. Sorry for the inconvenience 🙇

**Reply by zadamg** [reply]
Appreciate you.

**Reply by amyegan** [reply]
After a bit of testing, it looks like this bug is limited to some Hobby accounts (mine included). The team is actively investigating the issue and working to remove the bug. You can check the current status at https://www.vercel-status.com/incidents/2skd5q24g60b

**Reply by amyegan** [reply]
Confirmed it's working now 🎉

**Comment by franknoel** [comment]
I also have the same issue, but only with my hobby plan. The DNS screen of my team works fine.

**Comment by swarnava** [comment]
We have identified the root cause and working for a fix.

**Comment by swarnava** [comment]
This should be fixed now.

**Reply by zadamg** [reply]
'Tis fixed. Thank you!
